configs/base_config.py

```diff
@@ -93,6 +93,17 @@ def FLASKS3_CDN_DOMAIN(self):
     def FLASKS3_BUCKET_NAME(self):
         return self._get_config_value("FLASKS3_BUCKET_NAME")
 
+    @property
+    def OPEN_SEARCH_HOST(self):
+        return self._get_config_value("OPEN_SEARCH_HOST")
+
+    @property
+    def OPEN_SEARCH_HTTP_AUTH(self):
```

Comment:
this is fine for local dev but to get this working with AWS we will need to use iam so will need to override this method in 
`configs/aws_secrets_manager_config.py`

so that it uses something like

```
    from requests_aws4auth import AWS4Auth

    sts_client = boto3.client('sts')
    assumed_role = sts_client.assume_role(
        RoleArn=self._get_config_value("OPEN_SEARCH_MASTER_ROLE_ARN"),
        RoleSessionName="AYRWebappLambdaSession"
    )
    credentials = assumed_role['Credentials']
    open_search_http_auth = AWS4Auth(
        credentials['AccessKeyId'],
        credentials['SecretAccessKey'],
        self.AWS_REGION,
        'es',
        session_token=credentials['SessionToken']
    )
```

Reply:
we will test this on an aws deployment to mmake sure its working

Reply:
tested and works in AWS